rules/imports.js

```diff
@@ -0,0 +1,204 @@
+module.exports = {
+  env: {
```

Comment:
I don't think we want to add env and parserOptions here. 

`env` is handled in [`eslint-config-cup`](https://github.com/rtsao/create-universal-package/blob/master/packages/eslint-config-cup/eslintrc.js#L8).

`parseOptions` are handled in [`eslint-config-uber-base-stage-3`](https://github.com/uber-web/uber-eslint/blob/next/packages/eslint-config-uber-base-stage-3/index.js#L23).

Both of these are extended via `eslint-config-uber-universal-stage-3`.

Reply:
Done, thanks.